microsite/data/plugins/gke-usage.yaml

```diff
@@ -0,0 +1,15 @@
+---
+title: GKE Usage
+author: BESTSELLER
+authorUrl: bestsellerit.com
+category: Discovery
+description: This plugin will show you the cost and resource usage of your application within GKE.
+documentation: https://github.com/BESTSELLER/backstage-plugin-gkeusage/blob/master/README.md
+iconUrl: https://bestsellerit.com/img/google-container-engine_avatar.svg
+npmPackageName: "@bestsellerit/backstage-plugin-gkeusage"
```

Comment:
```suggestion
npmPackageName: '@bestsellerit/backstage-plugin-gkeusage'
```

Reply:
ok cool I was wondering why it was failing